ts/config.ts

```diff
@@ -241,8 +241,6 @@ export const itwWalletProviderBaseUrl = Config.ITW_WALLET_PROVIDER_BASE_URL;
 export const itwGoogleCloudProjectNumber =
   Config.ITW_GOOGLE_CLOUD_PROJECT_NUMBER;
 export const itWalletIssuanceRedirectUri = Config.ITW_ISSUANCE_REDIRECT_URI;
-export const itWalletIssuanceRedirectUriCie =
```

Comment:
Since it is no longer needed, should we also remove this from the env variables?

Reply:
You are right: removed in 1a647cc8cd28070f93332c468d2aeaf1cde2741f.